### PR TITLE
Add a "What's new" section to the Mod Docs standard

### PR DESCRIPTION
**Pull request by rolfedh**
Proposed fix for: Modular docs should have versioned releases #127
Fixes: #127

### CODE REVIEW

**Comment by rolfedh** [comment]
@emmurphy1

File: modular-docs-manual/content/topics/whats-new.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[id="whats-new_{context}"]
+= What's new
+
+The following list summarizes significant changes to the mod docs standard over time.
+In most cases, apply these changes to your new content and revisions.
```
Comment (ncbaratta) [inline modular-docs-manual/content/topics/whats-new.adoc:5]: I'm not sure what you're trying to say - with the next line I think you're saying you don't have to change anything in existing content, just new. Which means we don't need this line at all.

File: modular-docs-manual/content/topics/whats-new.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+It is not expected that you apply changes retrospectively to existing content.
+In rare cases where all content changes need to be made, the Mod Docs team will communicate this information through CCS leadership.
```
Comment (ncbaratta) [inline modular-docs-manual/content/topics/whats-new.adoc:7]:
```suggestion
CCS leadership will communicate in instances where changes to all content will need to be made.
```

File: modular-docs-manual/content/topics/whats-new.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+// Template:
```
Comment (ncbaratta) [inline modular-docs-manual/content/topics/whats-new.adoc:9]:
```suggestion
// Release Notes Template:
```

**Review by ncbaratta — CHANGES_REQUESTED** [review]

File: modular-docs-manual/content/topics/whats-new.adoc (hunk 1 above)
Comment (rolfedh) [reply modular-docs-manual/content/topics/whats-new.adoc:5]: Thanks. I hope the updated version is clearer.

File: modular-docs-manual/content/topics/whats-new.adoc (hunk 2 above)
Comment (rolfedh) [reply modular-docs-manual/content/topics/whats-new.adoc:7]: Thanks. Updated.

File: modular-docs-manual/content/topics/whats-new.adoc (hunk 3 above)
Comment (rolfedh) [reply modular-docs-manual/content/topics/whats-new.adoc:9]: Thanks. Updated.

**Review by rolfedh — COMMENTED** [review]
Thanks for your suggestions, @ncbaratta. I've updated the content accordingly.

File: modular-docs-manual/content/topics/whats-new.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[id="whats-new_{context}"]
+= What's new
+
+The following list summarizes significant changes to the mod docs standard over time.
+Apply these changes when you create and revise content.
```
Comment (ncbaratta) [inline modular-docs-manual/content/topics/whats-new.adoc:5]:
```suggestion
Apply these changes only when you create and revise content.
```

Added only so the next sentence makes sense, but am open to other suggestions.

Comment (rolfedh) [reply modular-docs-manual/content/topics/whats-new.adoc:5]: Thanks for the suggestion, @ncbaratta. Updated.

**Comment by rolfedh** [comment]
@ncbaratta to unblock this PR, please approve the changes.
image

**Review by ncbaratta — APPROVED** [review]

File: modular-docs-manual/content/topics/whats-new.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[id="whats-new_{context}"]
+= What's new
+
+The following list summarizes significant changes to the mod docs standard over time.
```
Comment (abhatt-rh) [inline modular-docs-manual/content/topics/whats-new.adoc:4]: _mod docs_ seems like an internal reference, consider using _modular documentation_ instead.
```suggestion
The following list summarizes significant changes to the modular documentation standard over time.
```

**Review by abhatt-rh — APPROVED** [review]
Thank you for this PR @rolfedh. I left a minor comment. Rest LGTM!

**Review by mjahoda — APPROVED** [review]
LGTM (with the same suggestion to change "mod docs" to "modular documentation" as Avani).

**Review by emmurphy1 — APPROVED** [review]

**Pull request merged**